paasta_tools/paasta_cli/cmds/validate.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import os
+import yaml
+
+from glob import glob
+
+from jsonschema import Draft4Validator
+from jsonschema import FormatChecker
+from jsonschema import ValidationError
+from paasta_tools.paasta_cli.utils import guess_service_name
+from paasta_tools.paasta_cli.utils import PaastaValidateMessages
+from paasta_tools.paasta_cli.utils import PaastaValidateSchemas
+
+
+def get_file_contents(path):
+    """Open a file for reading
+
+    :param path: path of file to read"""
+    return open(path).read()
```

Comment:
Sure. As an FYI, this function was from paasta_tools/paasta_cli/cmds/check.py . I guess I should probably extract it so we don't have 2 copies (and then both commands will properly close the file).